src/main/java/edu/cornell/kfs/module/purap/batch/dataaccess/impl/JaggaerUploadDaoJdbc.java

```diff
@@ -54,15 +60,17 @@ public List<JaggaerContractPartyUploadDto> findJaggaerContractParty(JaggaerContr
                 dto.setWebsiteURL(resultSet.getString(FIELD_NAMES.VNDR_URL_ADDR));
                 return dto;
             };
-            return this.getJdbcTemplate().query(buildVendorSql(processingMode, processingDate), rowMapper);
+            CuSqlQuery sqlQuery = buildVendorSql(processingMode, processingDate);
+            logSQL(sqlQuery);
```

Comment:
With your current setup, this query will always get logged at INFO level. Is that what you intended, or would it be better to log it at DEBUG level (or to at least place an "if" block to require this DAO to be at DEBUG level before invoking the method)?

Reply:
My intention is to always log the SQL for this particular batch job.  Logging the SQL will help diagnosis any issues.

Reply:
Ok, thanks for clarifying.